src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -43,8 +44,8 @@ public class CommandTestUtil {
     public static final String VALID_PRIORITY_BOB = "MEDIUM";
     public static final String VALID_REMARK_AMY = "Loves Taylor Swift.";
     public static final String VALID_REMARK_BOB = "Supports Manchester United.";
-    public static final String VALID_DATE_OF_BIRTH_AMY = "1 Jan 2000";
-    public static final String VALID_DATE_OF_BIRTH_BOB = "3 Jan 1989";
+    public static final LocalDate VALID_DATE_OF_BIRTH_AMY = LocalDate.of(2000, 1, 1);
+    public static final LocalDate VALID_DATE_OF_BIRTH_BOB = LocalDate.of(1989, 1, 3);
```

Comment:
Since `DateOfBirth` constructor only accepts argument of `LocalDate` type, I think it's better to keep them, rather than introduce additional parsing of date strings.

These static fields are also used for tests in `EditCommandParserTest`.